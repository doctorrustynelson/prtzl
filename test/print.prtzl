
print_number(3.1415926);
print_string("Hello, World!");

Vertex v = <+ "a" +>;

print_vertex(v);