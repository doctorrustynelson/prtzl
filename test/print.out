3.141593
Hello, World!
a: 0 edge(s) in 0 edge(s) out
